src/unit-tests/involvmentapi.test.js

```diff
@@ -0,0 +1,17 @@
+import { getLikes, postLike } from '../__mocks__/involvmentapi.js';
+
+jest.mock('../modules/involvmentapi.js');
+
+describe('Test for posting and getting a like(s) via the involvement api ', () => {
+  test('Posting a new like ', () => {
+    postLike({ item_id: '1' }).then((res) => {
+      expect(res).toBe('Created');
+    });
+  });
+
+  test('Getting the number of likes from the api ', () => {
+    getLikes().then((response) => {
+      expect(response.likes).toBe(5);
+    });
+  });
+});
```

Comment:
- [ ] Kindly work on the test on this codebase. Currently, you're having the following failing tests:

![image](https://user-images.githubusercontent.com/50941074/188269089-17553312-2850-476e-b197-c43e8f118df0.png)